server/package.json

```diff
@@ -73,8 +72,9 @@
     "@gladysassistant/gladys-gateway-js": "^3.2.6",
     "@hapi/joi": "^17.1.0",
     "@hapi/joi-date": "^2.0.1",
+    "axios": "^0.19.2",
```

Comment:
If you use specific dependency in your service, please add them to your service package.json

Reply:
this dependancy is not use in withings service but in server/lob/oauth2/executeQuery.js